packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  Component, NgModule, Input, SimpleChanges, OnChanges,
+} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HttpClientModule } from '@angular/common/http';
+import { Subscription } from 'rxjs';
+import {
+  DxGanttModule,
+  DxLoadPanelModule,
+} from 'devextreme-angular';
+import { Task } from 'src/app/shared/types/task';
+
+@Component({
+  selector: 'task-list-gantt',
+  templateUrl: './task-list-gantt.component.html',
+  styleUrls: ['./task-list-gantt.component.scss'],
+})
+export class TaskListGanttComponent implements OnChanges {
+  @Input() dataSource: Task[];
+
+  isLoading = true;
+
+  dataSubscription: Subscription = new Subscription();
```

Comment:
Why do you init dataSubscription? Looks like you don't use it in the component

Reply:
Fixed